internal/interfaceadapter/service/usersv/service.go

```diff
@@ -1 +1,48 @@
 package usersv
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/RettyInc/retty3-proto/go/messages"
+	"github.com/RettyInc/retty3-proto/go/services"
+	"github.com/RettyInc/retty3-service-user/internal/entity"
+)
+
+type UserUseCase interface {
+	GetUsers(ctx context.Context, ids []uint64) (map[uint64]*entity.User, error)
+}
+
+type service struct {
+	userUseCase UserUseCase
+}
+
+func New(userUseCase UserUseCase) *service {
+	return &service{
+		userUseCase: userUseCase,
+	}
+}
+
+func (s *service) GetUsers(ctx context.Context, query *services.UsersQuery) (*services.UsersResponse, error) {
+	userIDs := query.GetIds()
+	userIDToUser, err := s.userUseCase.GetUsers(ctx, userIDs)
```

Comment:
ユーザー情報が入ってるんだなーぐらいが伝われば良さそう
```suggestion
	userEntities, err := s.userUseCase.GetUsers(ctx, userIDs)
```

Reply:
こちらのほうが分かりやすいので修正しました！ありがとうございます！ 😄